src/rockstor/storageadmin/templates/storageadmin/base.html

```diff
@@ -100,6 +100,7 @@
     <script src="/static/js/lib/moment.min.js"></script>
     <script src="/static/js/lib/simple-slider.min.js"></script>
     <script src="/static/js/lib/chosen.jquery.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/0.9.6/socket.io.min.js"></script>
```

Comment:
This is merely me being lazy, @phillxnet. Once the code gets pushed to production, the hard copy of the file will be referenced.